.github/workflows/copr_build.yml

```diff
@@ -69,7 +69,7 @@ jobs:
       uses: next-actions/copr/filter-chroots@master
       with:
         coprcfg: ${{ steps.copr.outputs.coprcfg }}
-        filter: "fedora-.+-x86_64|centos-stream-9-x86_64"
+        filter: "fedora-.+-x86_64|centos-stream-9-x86_64|centos-stream-10-x86_64"
```

Comment:
You can also use `centos-stream-.*-x86_64` and add c8s to exclude list. If this was done in first place, c10s would be picked up automatically.

Reply:
Thanks, updated.